UnityProject/Assets/Scripts/UI/Systems/AdminTools/ServerSettingsPage.cs

```diff
@@ -1,31 +1,49 @@
 ﻿using AdminCommands;
 using TMPro;
 using UnityEngine;
+using UnityEngine.Serialization;
 
 namespace AdminTools
 {
 	public class ServerSettingsPage : AdminPage
 	{
+		[FormerlySerializedAs("inputField")] [SerializeField]
+		private TMP_InputField playerLimitInputField = null;
+
 		[SerializeField]
-		private TMP_InputField inputField = null;
+		private TMP_InputField serverMaxFrameRate = null;
 
 		public override void OnPageRefresh(AdminPageRefreshData adminPageData)
     	{
     		base.OnPageRefresh(adminPageData);
-            inputField.text = adminPageData.playerLimit.ToString();
+            playerLimitInputField.text = adminPageData.playerLimit.ToString();
+            serverMaxFrameRate.text = adminPageData.maxFrameRate.ToString();
         }
 
-		public void OnChangeInputField()
+		public void OnChangePlayerLimit()
 		{
-			if (int.TryParse(inputField.text, out var value))
+			if (int.TryParse(playerLimitInputField.text, out var value))
 			{
-				if (value < 0) 
+				if (value < 0)
 				{
 					value = 0;
 				}
-				
+
 				AdminCommandsManager.Instance.CmdChangePlayerLimit(value);
 			}
 		}
+
+		public void OnChangeFrameRate()
```

Comment:
only issue what happens if you relaog after seting the frame rate how do you get the updated value

Reply:
line 20

Reply:
cool